packages/api/src/mikro-orm/migrations/Migration20241119101706.ts

```diff
@@ -0,0 +1,7 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20241119101706 extends Migration {
+    async up(): Promise<void> {
+        this.addSql('alter table "BrevoConfig" add column "folderId" int not null default 1;');
```

Comment:
You need to drop the default afterwards again. Generally you should not use default values in the database. This is just useful to fill not nullable fields.

Reply:
515452df8cc7b6908f6e5f5d6966959774151771

Reply:
f6a6660efe183ff2248fd57ae5198e590cae382b